--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,7 @@
     set(GLSystem "Desktop OpenGL" CACHE STRING "The OpenGL system to be used")
 #-------------------------------------------------------------------------------
 #check if we're running on Raspberry Pi
-elseif(EXISTS "/opt/vc/include/bcm_host.h")
+elseif(EXISTS "${CMAKE_FIND_ROOT_PATH}/usr/include/bcm_host.h")
     MESSAGE("bcm_host.h found")
     set(BCMHOST found)
     set(GLSystem "OpenGL ES" CACHE STRING "The OpenGL system to be used")
@@ -32,7 +32,8 @@
     set(GLSystem "OpenGL ES" CACHE STRING "The OpenGL system to be used")
 #-------------------------------------------------------------------------------
 #check if we're running on olinuxino / odroid / etc
-elseif(EXISTS "/usr/lib/libMali.so" OR
+elseif(EXISTS "${CMAKE_FIND_ROOT_PATH}/usr/lib/libMali.so" OR
+    EXISTS "${CMAKE_FIND_ROOT_PATH}/usr/lib/libhybris-common.so" OR
     EXISTS "/usr/lib/arm-linux-gnueabihf/libMali.so" OR
     EXISTS "/usr/lib/aarch64-linux-gnu/libMali.so" OR
     EXISTS "/usr/lib/arm-linux-gnueabihf/mali-egl/libmali.so" OR
@@ -140,10 +141,10 @@
 
 if(DEFINED BCMHOST)
     LIST(APPEND COMMON_INCLUDE_DIRS
-        "/opt/vc/include"
-        "/opt/vc/include/interface/vcos"
-        "/opt/vc/include/interface/vmcs_host/linux"
-        "/opt/vc/include/interface/vcos/pthreads"
+        "${CMAKE_FIND_ROOT_PATH}/include"
+        "${CMAKE_FIND_ROOT_PATH}/include/interface/vcos"
+        "${CMAKE_FIND_ROOT_PATH}/include/interface/vmcs_host/linux"
+        "${CMAKE_FIND_ROOT_PATH}/include/interface/vcos/pthreads"
     )
 #add include directory for Vero4K
 elseif(DEFINED VERO4K)
@@ -166,7 +167,7 @@
 #define libraries and directories
 if(DEFINED BCMHOST)
     link_directories(
-        "/opt/vc/lib"
+        "${CMAKE_FIND_ROOT_PATH}/lib"
     )
 elseif(DEFINED VERO4K)
     link_directories(

