--- a/Makefile
+++ b/Makefile
@@ -137,6 +137,14 @@
     LDFLAGS += -L/opt/vero3/lib
     HAVE_NEON = 1
     NAME  = amiberry-vero4k
+
+else ifeq ($(PLATFORM),aml)
+USE_SDL2 = 1
+    CPU_FLAGS += -march=armv8-a -mtune=cortex-a53 -mfpu=neon-fp-armv8
+    CFLAGS += -I$(SYSROOT_PREFIX)/usr/include -DARMV6T2 -DUSE_ARMNEON -DARM_HAS_DIV -DUSE_SDL2 -DMALI_GPU -DUSE_RENDER_THREAD -DTINKER
+    LDFLAGS += -L$(SYSROOT_PREFIX)/usr/lib
+    HAVE_NEON = 1
+    NAME  = amiberry-aml-sdl2
 
 else ifeq ($(PLATFORM),tinker)
 USE_SDL2 = 1

